Web/Assignmrnt 2/myProfile.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>my profile</title>
+<link rel="stylesheet" type="text/css" href="mystyle.css">
+
+</head>
+<body>
+	<div class="wrapper">
+		<div class="header">My Profile</div>
+		<div class="image"></div>
+		<div class="info">
+			<table>
+				<tr>
+					<td colspan="2"><h3>Contact Information</h3></td>
+				</tr>
+				<tr>
+					<td>Email Id:</td>
```

Comment:
ACTION: Why is not input box used here? The whole page is a form as per the requirement and there is no input box whatsoever!